src/GovUk.Education.ExploreEducationStatistics.Data.Processor/ImportRecoveryHandler.cs

```diff
@@ -56,7 +56,7 @@ public static void CheckIncompleteImports(string connectionString)
                     Console.Out.WriteLine($"Recovering {entity.PartitionKey} : {entity.RowKey}");
 
                     // If batch was not split then just processing again by adding to pending queue
-                    if (entity.Status.Equals(IStatus.QUEUED) || entity.Status.Equals(IStatus.PROCESSING_ARCHIVE_FILE) || entity.Status.Equals(IStatus.RUNNING_PHASE_1))
+                    if (entity.Status.Equals(IStatus.QUEUED) || entity.Status.Equals(IStatus.PROCESSING_ARCHIVE_FILE) || entity.Status.Equals(IStatus.STAGE_1))
```

Comment:
The conditional is getting a bit long, could improve readability by putting each part of the conditional on a new line

Reply:
Separated onto new lines